src/manifolds/ProbabilitySimplex.jl

```diff
@@ -8,8 +8,8 @@ The (relative interior of) the probability simplex is the set
 ````
 where $\mathbb{1}=(1,…,1)^{\mathrm{T}}∈ ℝ^{n+1}$ denotes the vector containing only ones.
 
-If `closed` is set to `false`, then the object represents an open simplex. Otherwise,
-that is when `closed` is set to `true`, the boundary is also included:
+If `closed` is set to `:open`, then the object represents an open simplex. Otherwise,
+that is when `closed` is set to `:Closed`, the boundary is also included:
```

Comment:
should we capitalise the symbols or not?

Reply:
I'm not sure, I think non-capitalized is more common in Julia but I'm fine with either choice.